students/sikorski_marcin/Lesson_6/list_to_dictionary_function_for_fantasy_game_inventory.py

```diff
@@ -0,0 +1,28 @@
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+dragonLoot = ['gold coin', 'dagger', 'gold coin', 'gold coin', 'ruby']
+
+
+def display_inventory(inventory):
+    print("Inventory: ")
+    item_total = 0
+    for key, value in inventory.items():
+        print(str(value) + ' ' + str(key))
+        item_total += value
+    print("Total number of items: " + str(item_total))
+
+display_inventory(stuff)
+
+
+def add_to_inventory(inventory, added_items):
+    for i in range(len(added_items)):
```

Comment:
tutaj możesz wykorzystac bezposredniej iteracji po elementach sekwencji:
` for item in added_items:`
`    inventory.setdefault(item, 0)`